.github/workflows/beta-deploy.yml

```diff
@@ -0,0 +1,51 @@
+name: Beta Deployment Workflow
+
+on:
+  issue_comment:
+    types: [created]
```

Comment:
We want to deploy on merge from main or dispatch action. Can you please elaborate on this trigger?

Reply:
The point of approval workflow is to add a manual step between PR merge and Deployment of the changes from the PR to Beta followed by Prod. 
The changes in the current PR will
1. Triggers a changeset workflow to review the resources that changed/added/deleted in both beta and prod environments.
2. After the PR is merged, folks from the team would again confirm the changeset and comment "Approve ...." sequentially to procced beta and prod deployment.